server/server.ts

```diff
@@ -154,8 +166,9 @@ app.get('/api/entries/:entryId', async (req, res, next) => {
       throw new ClientError(400, `${entryId} does not exist`);
     }
     const sql = `
-      select *
+      select "entryId", "userId", "title", "subtitle", "location", "body", "date", "photoURL", "photoAlt", "photoAuthor", "photoAuthorLink", "userId", "username"
```

Comment:
I didn't use * because it was returning the hashed passwords from the db because I'm joining the users table with the entries table. Because it is hashed I assume its not a big deal if I am returning it but just omitted it incase it was a problem.